### PR TITLE
Trio bug fixes, added NTPD agent, Fixed naming issues with the NTPD and MERC PDAs, Added admin stuff.

### PR DESCRIPTION
**Pull request by Guloveos**
Description.

Fixed a bug in some of the trios code where the would gib before reaching half health.

Added the NTPD agent, A stealth oriented NTPD role to complete objectives.

fixed naming issues with the NTPD and MERC PDAs going from "NTPD, PDA" to "NTPD PDA".

Added admin references.

:cl:
- add: Added NTPD agent, admin reference items.
- fix: Fixed PDA's having the wrong name, Fixed trio code to not kill before reaching half health.

### CODE REVIEW

File: Resources/Prototypes/Entities/Arcadis_station/Admin.yml [hunk 1]
```diff
@@ -812,3 +814,141 @@
...
   - type: Sprite
     state: FBI
+
+#seths gun
+
+- type: entity
+  name: Dead Again Hand Cannon
+  parent: BaseWeaponLauncher
+  id: WeaponLaunchersethCannon
+  description: so strong it kills your enemies twice!
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Arcadis_station/Admin.yml:824]:
```suggestion
  description: So strong it kills your enemies twice!
```

File: Resources/Prototypes/Entities/Arcadis_station/Admin.yml [hunk 2]
```diff
@@ -812,3 +814,141 @@
   components:
   - type: Sprite
     state: FBI
+
+#seths gun
+
+- type: entity
+  name: Dead Again Hand Cannon
+  parent: BaseWeaponLauncher
+  id: WeaponLaunchersethCannon
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Arcadis_station/Admin.yml:823]:
```suggestion
  id: WeaponLauncherSethCannon
```

File: Resources/Prototypes/Entities/Objects/Devices/pda.yml [hunk 3]
```diff
@@ -1397,7 +1397,7 @@
 - type: entity
   parent: BasePDA
   id: NTPDPDA
-  name: NTPD, PDA
+  name: NTPD PDA
   description: an NTPD PDA, now say that five times fast.
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Objects/Devices/pda.yml:1401]:
```suggestion
  description: An NTPD PDA, now say that five times fast.
```

**Comment by Erisfiregamer1** [comment]
Speedmerge time since an event is incoming. Federal's gonna HATE this one!

File: Resources/Prototypes/Entities/Objects/Devices/pda.yml [hunk 4]
```diff
@@ -1471,7 +1471,7 @@
 - type: entity
   parent: BasePDA
   id: MERCPDA
-  name: merc, PDA
+  name: merc PDA
   description: a merc PDA. Urban Solutions standerd.
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Objects/Devices/pda.yml:1475]:
```suggestion
  description: A standard issue Urban Solutions PDA. Smells faintly of gunpowder.
```

**Review by TheDoctor1977 — CHANGES_REQUESTED** [review]

**Comment by TheDoctor1977** [comment]
> Speedmerge time since an event is incoming. Federal's gonna HATE this one!

i’ll kill you for this

**Comment by Erisfiregamer1** [comment]
> > Speedmerge time since an event is incoming. Federal's gonna HATE this one!
> 
> i’ll kill you for this

make the pr with changes trolley

**Comment by TheDoctor1977** [comment]
> > > Speedmerge time since an event is incoming. Federal's gonna HATE this one!
> > 
> > 
> > i’ll kill you for this
> 
> make the pr with changes trolley

after the event just revert, make changes, remerge
i’m sick and don’t feel like making a new PR

**Comment by Erisfiregamer1** [comment]
if gul accepts the changes they can just make another PR from github as it'll create a new commit

**Pull request merged**